tests/client_tests/stock_jdbc/src/test/java/stock_jdbc/JdbcMetaDataTest.java

```diff
@@ -551,7 +551,7 @@ public void test_getTables() throws Exception {
         try (var conn = DriverManager.getConnection(URL)) {
             var results = conn.getMetaData().getTables(null, "sys", "", null);
             assertThat(results.next(), is(true));
-            assertThat(results.getString(3), is("allocations"));
+            assertThat(results.getString(3), is("allocations_pkey"));
```

Comment:
Do we have an issue somewhere in CrateDB that `getTables` returns a `_pkey` entry here? That doesn't seem right

Reply:
Ok, i was not sure about this. This just happened once i updated to a more recent version. I will have  a look into it.